docs/source/upgrade.rst

```diff
@@ -616,13 +705,13 @@ Compatibility Matrix
      - Docs
    * - v14
      - 3.5~
```

Comment:
Minimum CC supported is now 5.0 as we dropped CUDA 11.

```suggestion
     - 5.0~
```
 https://github.com/cupy/cupy/blob/d99df5b322d948b85bc1da486da893ad56fe6c77/install/cupy_builder/_compiler.py#L102